### DISCUSSION TITLE
Error raised when using numba, class Parameters need @jit predefined

### DISCUSSION THREAD

**Discussion by Eririf** [lmfit/lmfit-py#916] [Q&A]
TypingError: Failed in nopython mode pipeline (step: nopython frontend)
non-precise type pyobject
During: typing of argument at /tmp/ipykernel_1593/2842184324.py (9)

File "../../../../../../tmp/ipykernel_1593/2842184324.py", line 9:
<source missing, REPL/exec in use?> 

This error may have been caused by the following argument(s):
- argument 2: Cannot determine Numba type of <class 'lmfit.parameter.Parameters'>

**Comment by newville** [comment]
@Eririf  I am not surprised that trying to use numba with lmfit would cause problems.  But you have not given even a hint of what you did and you deliberately edited and truncated whatever error messages you got.  I cannot guess what you expect us to do with this message.
